gpflow/models/gplvm.py

```diff
@@ -75,10 +76,10 @@ def __init__(
         super().__init__(gpr_data, kernel, mean_function=mean_function)
 
 
-class BayesianGPLVM(GPModel):
+class BayesianGPLVM(GPModel, InternalDataTrainingLossMixin):
     def __init__(
         self,
-        data: tf.Tensor,
+        data: OutputData,
```

Comment:
what's the difference between input and output data? it feels odd to be passing in output data to a class

Reply:
Y = f(X) <-- X input, Y output; here we're observing Y but don't know X, so the model only gets the output. (and for point process models, there is no Y, just X.)